accord-core/src/main/java/accord/topology/TopologyManager.java

```diff
@@ -61,10 +63,10 @@ public class TopologyManager implements ConfigurationService.Listener
         private boolean syncComplete = false;
         private boolean prevSynced;
 
-        EpochState(Node.Id node, Topology global, boolean prevSynced)
+        EpochState(Id node, Topology global, boolean prevSynced)
         {
             this.global = global;
-            this.local = global.forNode(node);
+            this.local = global.forNode(node).trim();
```

Comment:
Should trimming need to be done manually?

Reply:
I think so, yes, since it is rarely needed.